> dont merge without tests

Running ```make acceptance``` somehow doesn't work

```
linux@ecs-cse-jumphost:~/otc_terraformprovider_dev/terraform-provider-opentelekomcloud$ make acceptance                   ==> Checking that code complies with gofmt requirements...

Starting acceptance tests...
Checking diff with `devel`
The following services needs testing:
cce
malformed import path "github.com/opentelekomcloud/terraform-provider-opentelekomcloud/opentelekomcloud/acceptance/cce\ngithub.com/opentelekomcloud/terraform-provider-opentelekomcloud/opentelekomcloud/acceptance/cce/shared": invalid char '\n'
make: *** [GNUmakefile:23: acceptance] Error 1
```

--- a/opentelekomcloud/services/cce/resource_opentelekomcloud_cce_addon_v3.go
+++ b/opentelekomcloud/services/cce/resource_opentelekomcloud_cce_addon_v3.go
@@ -191,6 +191,10 @@ func resourceCCEAddonV3Delete(ctx context.Context, d *schema.ResourceData, meta 
 
 	clusterID := d.Get("cluster_id").(string)
 
+	if err := addons.Delete(client, d.Id(), clusterID).ExtractErr(); err != nil {
+		return fmterr.Errorf("error deleting CCE addon : %w", err)
+	}
+
 	stateConf := &resource.StateChangeConf{
 		Pending:      []string{"available"},
 		Target:       []string{"deleted"},
@@ -248,13 +252,6 @@ func unStringMap(src map[string]interface{}) map[string]interface{} {
 
 func waitForCCEAddonDelete(client *golangsdk.ServiceClient, addonID, clusterID string) resource.StateRefreshFunc {
 	return func() (interface{}, string, error) {
-		if err := addons.Delete(client, addonID, clusterID).ExtractErr(); err != nil {
-			if _, ok := err.(golangsdk.ErrDefault404); ok {
-				return nil, "deleted", nil
-			}
-			return nil, "error", fmt.Errorf("error deleting CCE addon : %w", err)
-		}
-
 		addon, err := addons.Get(client, addonID, clusterID).Extract()
 		if err != nil {
 			if _, ok := err.(golangsdk.ErrDefault404); ok {

--- a/releasenotes/notes/cce-poll-interval-41899411a85a9039.yaml
+++ b/releasenotes/notes/cce-poll-interval-41899411a85a9039.yaml
@@ -1,0 +1,5 @@
+---
+
+enhancements:
+  - |
+    **[CCE]** Add ``PollInterval`` in ``resourceCCEAddonV3Delete`` in ``resource/opentelekomcloud_cce_addon_v3`` (`#1810 <https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/pull/1810>`_)

